### PR TITLE
HelmRelease with spec YAML 

### PR DESCRIPTION
**Pull request by pezhang**
1. Add the YAML file with HelmRelease specifying spec field
2. This YAML file will be used for testing " A normal sunny deployment with some spec defined"

### CODE REVIEW

**Comment by pezhang** [comment]
@ianzhang366 Could you help to review this patch? Thanks a lot.

File: examples/test-guestbook-spec.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  name: guestbook
+  namespace: helmrelease-spec-test
+repo:
+  chartName: nginx-chart
+  source:
+    github:
+      branch: gh-pages
+      chartPath: nginx-chart
+      urls:
+        - https://github.com/pezhang/helm-release-API-test.git
```
Comment (mikeshng) [inline examples/test-guestbook-spec.yaml:19]: I don't think we should point at personal repo like this.

Could you not upload the charts you need to this repo first then reference this repo?

**Comment by pezhang** [comment]
@mikeshng  Thank you for the review and approve.

**Pull request merged**